src/Handler/Setup.php

```diff
@@ -27,13 +30,8 @@ class Setup
     private $dependencies;
     private $options;
 
-    public function __construct(
-        string $cwd,
-        AppName $appName,
-        EnvEnum $envName,
-        \Traversable $dependencies,
-        array $options = []
-    ) {
+    public function __construct(string $cwd, AppName $appName, EnvEnum $envName, \Traversable $dependencies, array $options = [])
```

Comment:
Should we shop down methods' arguments if too long, or not ? (We're constantly changing this 😝 )